packages/cli/test/unit/commands/git/connect.test.ts

```diff
@@ -126,10 +131,6 @@ describe('git connect', () => {
             key: 'flag:yes',
```

Comment:
Not quite related to this PR, but isn't it weird that `flag:yes` is tracked, even though the user didn't explicitly type `--yes`?

Reply:
@TooTallNate it _is_ yes. And means we can easily know if `--confirm` is used but the query to know if `--yes` was used directly or as a side effect of passing `--confirm` is more complicated. 

There was a discussion somewhere about fixing this but IIRC we decided that since we had no tests (yet), changing the behavior had a high risk of breaking behavior.

Now that we have tests I think we can swing back through and fix this so you only see one or the other in metrics. Whether that blocks shipping or not I'll leave up to @erikareads

Reply:
Yeah, we'd have to move tracking of both up into the outer command. I'll change that.